benchmarks/dungeon-benchmark/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+plugins {
+  id("com.android.test")
+  id("org.jetbrains.kotlin.android")
+}
+
+// Note: We are not including our defaults from .buildscript as we have different min and do
+// not need the base Workflow dependencies.
+
+android {
+  compileSdk = 32
+
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
+
+  kotlinOptions {
+    jvmTarget = "1.8"
+  }
+
+  defaultConfig {
+    minSdk = 23
+    targetSdk = 32
+
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+  }
+
+  buildTypes {
+    create("benchmark") {
+      isDebuggable = true
```

Comment:
You likely want profileable but not debuggable? Or is that not the main app, but the app that runs the test instead?

Reply:
This is the app that runs the test.